src/controls/SelectTeamChannelPicker/ISelectTeamChannelPickerProps.ts

```diff
@@ -0,0 +1,13 @@
+import { WebPartContext } from "@microsoft/sp-webpart-base";
+import { ExtensionContext } from "@microsoft/sp-extension-base";
+import { IBasePickerStyles, IPickerItemProps, ISuggestionItemProps, ITag } from "office-ui-fabric-react";
+export interface ISelectTeamChannelPickerProps {
+  teamId:string | number;
+  appcontext:  WebPartContext |  ExtensionContext;
```

Comment:
We can use `BaseComponentContext` instead of `WebPartContext | ExtensionContext`.